calcom/cal.com
# fix: onboard-pricing-options-self-host

## What does this PR do?

- Change $99 plan with a link to contact sales page.

Fixes #10075 
## Requirement/Documentation

<img width="855" alt="image" src="https://github.com/calcom/cal.com/assets/4461358/470174e8-f401-4f32-a0b6-f12c6042857b">


## Type of change


## How should this be tested?

  - Are there environment variables that should be set? No.
  - What are the minimal test data to have? DB without any user admin created.
  - What is expected (happy path) to have (input and output)? 
  Button should redirect to cal.com/sales pages instead of being selectable.
  - Any other important info that could help to test that PR
  Modifying/Commenting code on the  SSR function that returns 404 when an user admin exist on the DB on auth/index so auth/setup doesn't return 404

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
